app/controllers/builds_controller.rb

```diff
@@ -98,6 +106,12 @@ def start_docker_build
     DockerBuilderService.new(@build).run!(push: true)
   end
 
+  def find_kuber_releases
+    if @build.respond_to?(:kubernetes_releases)
+      @kuber_release_list = @build.kubernetes_releases.order('id desc')
+    end
+  end
```

Comment:
Are any of these still used? It looks like you have a KubernetesReleasesController